.gitignore

```diff
@@ -21,3 +21,4 @@ __pycache__
 .sconsign.dblite
 /dbt_options.py
 /scripts/debug/openocd.log
+.vscode/*
```

Comment:
remove this file from your commit (git rm --cached I think)

Reply:
Werps! That was from merging my own community into this branch to get up to date. So what's the best way for me to keep this line locally but not have to keep excluding it from pushes?